stl/inc/atomic

```diff
@@ -728,7 +728,7 @@ struct _Atomic_storage<_Ty, 8> { // lock-free using 8-byte intrinsics
         return reinterpret_cast<_Ty&>(_As_bytes);
     }
 
-#ifdef _M_IX86
+#if defined(_M_IX86) && defined(__clang__)

```

Comment:
Ideally, we should file a Clang bug/feature request for parity with MSVC here, and mark this (and the second occurrence) as TRANSITION. If you're unfamiliar with the process (or don't want to manually request an account at [bugs.llvm.org](https://bugs.llvm.org/); hopefully they'll use GitHub issues soon), let us know and a maintainer can handle it.

Reply:
I've started the process of registration.
I see `// TRANSITION, LLVM-38217` and others, so will try to create an issue and follow the pattern.